From ee04e0c4c3e6b0dcb4379ff1a098a9f690949175 Mon Sep 17 00:00:00 2001
From: Jordan Esh <jordan.esh@monashmotorsport.com>
Date: Thu, 30 Dec 2022 21:35:41 +0200
Subject: [PATCH 01] stm32ecu make

---
 drivers/Kconfig    | 1 ++
 drivers/Makefile   | 1 ++
 2 files changed, 2 insertions(+)

diff --git a/drivers/Kconfig b/drivers/Kconfig
index 57699bd8f..d1f79bc2c 100644
--- a/drivers/Kconfig
+++ b/drivers/Kconfig
@@ -235,4 +235,6 @@
 source "drivers/counter/Kconfig"

 source "drivers/most/Kconfig"
+
+source "drivers/stm32ecu/Kconfig"
 endmenu
diff --git a/drivers/Makefile b/drivers/Makefile
index 57699bd8f..d1f79bc2c 100644
--- a/drivers/Makefile
+++ b/drivers/Makefile
@@ -189,3 +189,4 @@
 obj-$(CONFIG_INTERCONNECT)	+= interconnect/
 obj-$(CONFIG_COUNTER)		+= counter/
 obj-$(CONFIG_MOST)		+= most/
+obj-m					+= stm32ecu/
--
2.25.1
